### PR TITLE
Added docs for HDRi Skybox component

### PR DESCRIPTION
**Pull request by chanmosq**
### Submission Checklist:

* [X] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [X] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [X] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [X] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/components/reference/atom/hdri-skybox.md [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
-Add properties.
-{{< /todo >}}
+![HDRi Skybox interface](/images/user-guide/components/reference/atom/hdri-skybox-component-ui.png)
+
+
+### HDRi Skybox properties
+
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Cubemap Texture** | A *cubemap texture* is a collection of six 2D images. Each image is a square of equal size and maps to a face of a cube. The cubemap represents the sky in your scene at every direction of the world axes. | A `.streamingimage` product asset. |  |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/hdri-skybox.md:30]:
```suggestion
| **Cubemap Texture** | A *cubemap texture* is a collection of six 2D images. Each image is a square of equal size and maps to a face of a cube. The cubemap represents the sky in your scene at every direction of the world axes. | A `.streamingimage` product asset. | None |
```

**Review by FiniteStateGit — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/components/reference/atom/hdri-skybox.md [hunk 2]
```diff
@@ -1,20 +1,38 @@
...
+| **Exposure** | Exponentially scales the intensity of the exposure. | Float: -20.0 - 20.0 | `0.0` |
```
Comment (HogJonny-AMZN) [inline content/docs/user-guide/components/reference/atom/hdri-skybox.md:31]: HDRi Skybox exposure value unit is a EV100 scale (a Log2 exponential luminance and exposure value table used in photography and film), which is an [Exposure Value (EV)](https://en.wikipedia.org/wiki/Exposure_value) with a 100 International Standards Organization (ISO).  Within the Atom rendering pipeline, the skybox pixel intensity is multiplied by the EV100 luminance value to generate a HDR value to be rendered. The final on screen HDR values will be altered by render passes that occur later in the rendering pipeline, such as the 'Exposure Component' (a fullscreen postfx pass), and then the Display Mapper which will tonemap the pixel and apply the ODT (output display transform) resulting in final value to be rendered on screen (to either a LDR or HDR monitor.)

**Review by HogJonny-AMZN — CHANGES_REQUESTED** [review]
Some suggestions to expand education on terms as well as some instructions on how to actually go about generating the correct inputs for the component.

**Comment by FiniteStateGit** [comment]
Additional material: https://github.com/o3de/o3de/wiki/%5BHow-To%5D-Work-with-Skybox-and-Skylight-Configurations

**Review by micronAMZN — APPROVED** [review]
Approving pending feedback from @FiniteStateGit

**Review by FiniteStateGit — APPROVED** [review]
My suggestion is non-blocking, approving.

**Pull request merged**